db.go

```diff
@@ -248,25 +297,20 @@ func (db *DB) send(method string, params ...interface{}) (interface{}, error) {
 
 // resp is a helper method for parsing the response from a query.
 func (db *DB) resp(_ string, params []interface{}, res interface{}) (interface{}, error) {
-	arg, ok := params[0].(string)
-
-	if !ok {
-		return res, nil
-	}
+	if arg, ok := params[0].(string); ok {
+		if strings.Contains(arg, ":") {
```

Comment:
Yes, before PR implementation just checks if it is an array or not.
It is not an actual permission check that's why I replaced it because if it is not empty row not exists or it can't find.

Reply:
Can we delete `PermissionError` in that case if it isn't used?